spec/services/course/user_invitation_service_spec.rb

```diff
@@ -243,6 +243,24 @@ def invite
           expect(subject.flatten.count).to eq(1)
         end
       end
+
+      context 'when the provided csv file has no header' do
```

Comment:
That's handled in the existing test for the file with blanks, in the context above this one.